docs-source/content/quickstart/install.md

```diff
@@ -26,8 +26,23 @@ EOF
 
 #### Create a Traefik (Ingress-based) load balancer.
 
+Create a namespace of the load balancer
+
+```bash
+$ kubectl create namespace traefik
+
+```
+
 Use `helm` to install the [Traefik](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/README.md) load balancer. Use the [values.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/values.yaml) in the sample but set `kubernetes.namespaces` specifically.
 
+For helm version 2.x, you should have initialized if it has not been done already.
+
+```bash
+$ helm init
+```
```

Comment:
Helm is also used to install the operator, should the same instructions be included there?

Reply:
I added a new section in the beginning Setup Helm.  There isn't any setup before but Helm 3 has to setup the repo manually first, so just created a new section